src/features/auth/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from "./store";
-export * from "./components";
+export * from "./components/signin";
```

Comment:
why did you move these components ?

Reply:
or i must make sigin and signup in different component directory ?

Reply:
You should have them in the same directory,
the `components` directory of the `auth` feature